.vitepress/config.ts

```diff
@@ -664,33 +610,99 @@ function sidebarHome() {
                         },
                       ],
                     },
+                    
                   ],
                 },
+                {
+                  text: "Ethereum fallback mechanism",
+                  link: "/developers/ethereum-fallback",
+                },
               ],
             },
-            { text: "Rollkit", link: "/developers/rollkit" },
             {
-              text: "Astria",
+              text: "Sovereign",
+              collapsed: true,
+              items: [
+              {
+                text: "Dymension",
+                link: "https://dymension.xyz/",
+              },
```

Comment:
They are not sovereign as they settle on dymension's hub. 
let's make the category others for now and put dymension there

Reply:
got it. will make the change. what about astria?

Reply:
https://github.com/celestiaorg/docs/pull/1515/commits/c6e53a38ee4fc6cef432c08547899907137e1169